website/client/js/services/userServices.js

```diff
@@ -100,11 +100,13 @@ angular.module('habitrpg')
               }
             });
           }
-
-          return Tasks.getUserTasks();
+          let completedTodos = Tasks.getUserTasks(true);
+          let nonCompletedTasks = Tasks.getUserTasks();
+          return Promise.all([completedTodos, nonCompletedTasks]);
```

Comment:
Native promises are not supported in every browser we support: http://caniuse.com/#feat=promises